onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -863,6 +863,14 @@ def _where_input_wrangler(
         matcher=lambda sample: not (sample.args[0][0].dtype == torch.int64),
         reason="this Aten overload only supports tensor(int) as indices",
     )
+    TorchLibOpInfo(
+        "_unsafe_index_put",
```

Comment:
Looks like _unsafe_index_put does not exist in  https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py so we may need to create our own test cases.

Since it looks like it is the same as index_put, we can actually make it the same function